### PR TITLE
Check if syntax tree is present

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

File: src/Features/Core/Portable/Completion/AbstractCompletionService.cs [hunk 1]
```diff
@@ -278,17 +278,20 @@ private static CompletionItem Disambiguate(CompletionItem item, CompletionItem e
             CancellationToken cancellationToken)
         {
             var context = new CompletionListContext(document, position, triggerInfo, options, cancellationToken);
-            var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
-            if (!root.FullSpan.IntersectsWith(position))
+            if (document.SupportsSyntaxTree)
```
Comment (Pilchie) [inline src/Features/Core/Portable/Completion/AbstractCompletionService.cs]: You will still need line 299 even if this check succeeds.

**Comment by Pilchie** [comment]
:-1:

**Comment by jmarolf** [comment]
test vsi please

**Comment by jmarolf** [comment]
test vsi please

**Comment by jmarolf** [comment]
@dotnet/roslyn-ide

**Comment by jmarolf** [comment]
test vsi please

**Comment by jasonmalinowski** [comment]
:+1: if everything passes.

**Pull request merged**